### ISSUE TITLE
different spotspacing optimization results

### ISSUE THREAD

**Issue by Phoebegit** [e0404/matRad#322]
Hello,
    There is a question about optimization algorithm that has been bothering me.
    I used BoxPhantom.mat to do the single field plan with 270 gantry angle & 0 couch angle . The machine data is about our own machine,the bixel width is 7mm or 8mm. But the result is strange . the result of 8mm bixel width is better than the result of 7mm compared from DVH & objective value. The picture shows the DVH of these two result.
![image](https://user-images.githubusercontent.com/31639780/50519354-78b67700-0af5-11e9-83ee-9f7160f1f9f5.png)

 In my  opinion, the bigger spot spacing ,the little spot number,So the Solution space may be more small for bigger spot spacing. the possibility of 8mm bixel width of getting better result is smaller than 7mm . But the calculation result is reverse.
   Is this because the solution is a local solution?I changed the initial weight of 7mm bixel width plan , the result is also worse than 8mm. 
   So I don't know why? Or my opinion is wrong? It is OK that  the result of 7mm bixel width is worse than 8mm bixel width?

Your Sincerely!

Phoebe

**Comment by markbangert** [comment]
Hi Phoebe, we are all on holidays until 7 January...

Meanwhile here is a short answer: going from 8 to 7mm might have all sorts of discretization effects that could explain the behaviour that you are seeing, e.g. the spot placement just on the border around the target. Also premature convergence might be an issue with the result. You could try running the optimization with tighter settings.

If you are interested in systematically studying these effects I would also do runs with 6, 5, 4, 3mm and see what's coming out.

Cheers, Mark

**Comment by Phoebegit** [comment]
Wow,Enjoy your holidays!
  Discussing this problem after you vacation may be a better choice.
And thanks for your reply!

**Comment by markbangert** [comment]
Any update on this issue from your side? Did you run tests with a finer granularity?

**Comment by Phoebegit** [comment]
Yes,I used BOXPHANTOM.mat to do the signal field plan with bixel from 3mm to 9mm, The original result DVH comparison is shown in this picture
![image](https://user-images.githubusercontent.com/31639780/50735080-c90f9200-11e3-11e9-81d6-8d076afb0b67.png)



Zooming up target of the figure
![image](https://user-images.githubusercontent.com/31639780/50735078-b72def00-11e3-11e9-89d0-e71664077037.png)


This picture shows 8mm spot spacing plan is better than 5,6,7,8,9 spot spacing plan,except 8mm bixel, the other bixel's plan is almost keep the law: the bigger the spot spacing, the worse the plan


Then I created a water box ,the size is 10*10*10cm^3 . I also used this phantom to do the plans with bixel from 3mm to 9mm, The result shown
![image](https://user-images.githubusercontent.com/31639780/50735073-96fe3000-11e3-11e9-8cbc-bde27746fe6c.png)


Zooming up target part of this figure
![image](https://user-images.githubusercontent.com/31639780/50735069-80f06f80-11e3-11e9-9503-dcc7ef294980.png)



The DVH of 4mm, 6mm is worse than 5mm , 7mm respectively. 
The result of 10*10*10cm^3 phantom is different with the BOXPHANTOM.mat plan result.


Maybe Just like what you said ,There are many factors influence the dose result beside bixel, which cuased the confused result.

Thanks for your reply very much!
